flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -388,6 +409,20 @@
         DROP
     }
 
+    /**
+     * The enforcer to guarantee that precision of CHAR/VARCHAR columns is respected when writing
+     * data into sink.
+     */
+    public enum CharPrecisionEnforcer {
+        /**
+         * Throws runtime exception when writing string values exceeding the length defined in the
+         * type .
+         */
+        ERROR,
+        /** Trip string values to match the length defined as the CHAR/VARCHAR precision. */
```

Comment:
```suggestion
        /** Trim string values to match the length defined as the CHAR/VARCHAR precision. */
```

Reply:
Probably Trims (with s at the end) as `Throws`  for `ERROR`